tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -279,6 +279,13 @@ sub prepare_guest_for_sriov_passthrough {
         script_run "ssh root\@$vm 'systemctl restart systemd-journald'";
     }
 
+    # Release DHCP client IP before quit
+    my $dhcp_conf_file = "/etc/sysconfig/network/dhcp";
+    if ((script_run "ssh root\@$vm \"ls $dhcp_conf_file\"") == 0) {
+        script_run "ssh root\@$vm \"sed -i '/^[# ]*DHCLIENT_RELEASE_BEFORE_QUIT *=/{h;s/^[# ]*DHCLIENT_RELEASE_BEFORE_QUIT *=.*\\\$/DHCLIENT_RELEASE_BEFORE_QUIT=\\\"yes\\\"/};\\\${x;/^\\\$/{s//DHCLIENT_RELEASE_BEFORE_QUIT=\\\"yes\\\"/;H};x}' $dhcp_conf_file\"";
+        script_run "ssh root\@$vm \"grep 'DHCLIENT_RELEASE_BEFORE_QUIT' $dhcp_conf_file\"";
+    }
```

Comment:
I do not know the whole story well, but here it confuses me a little bit for using script_run instead of assert_script_run. Any reasons to abandon the certainty by assert? What if the $dhcp_conf_file does not exist? Will the test go well without modification?

Reply:
I just coped from the enabling debug logging part and did not think much. Yes, the configuration change is mandatory rather than optional. I'll make changes.